iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePageInfoData.mm

```diff
@@ -83,9 +83,34 @@ - (instancetype)initWithRawData:(Info const &)rawData ohLocalization:(id<IOpenin
           if (value == "yes")
             _driveThrough = NSLocalizedString(@"drive_through", nil);
           break;
-
-        default:
-          break;
+        case MetadataID::FMD_WEBSITE_MENU: _websiteMenu = ToNSString(value); break;
+        case MetadataID::FMD_CUISINE: break;
+        case MetadataID::FMD_FAX_NUMBER: break;
+        case MetadataID::FMD_STARS: break;
+        case MetadataID::FMD_ELE: break;
+        case MetadataID::FMD_TURN_LANES: break;
+        case MetadataID::FMD_TURN_LANES_FORWARD: break;
+        case MetadataID::FMD_TURN_LANES_BACKWARD: break;
+        case MetadataID::FMD_POSTCODE: break;
+        case MetadataID::FMD_DESCRIPTION: break;
+        case MetadataID::FMD_FLATS: break;
+        case MetadataID::FMD_HEIGHT: break;
+        case MetadataID::FMD_MIN_HEIGHT: break;
+        case MetadataID::FMD_DENOMINATION: break;
+        case MetadataID::FMD_BUILDING_LEVELS: break;
+        case MetadataID::FMD_TEST_ID: break;
+        case MetadataID::FMD_CUSTOM_IDS: break;
+        case MetadataID::FMD_PRICE_RATES: break;
+        case MetadataID::FMD_RATINGS: break;
+        case MetadataID::FMD_AIRPORT_IATA: break;
+        case MetadataID::FMD_BRAND: break;
+        case MetadataID::FMD_DURATION: break;
+        case MetadataID::FMD_DESTINATION: break;
+        case MetadataID::FMD_DESTINATION_REF: break;
+        case MetadataID::FMD_JUNCTION_REF: break;
+        case MetadataID::FMD_BUILDING_MIN_LEVEL: break;
+        case MetadataID::FMD_LOCAL_REF: break;
+        case MetadataID::FMD_COUNT: break;
```

Comment:
What for these cases? They don't bring any value comparing to default. Paranoid ObjC warning?

Reply:
Yes it's only for warnings 

https://github.com/organicmaps/organicmaps/pull/8244#discussion_r1608849200


I agree that empty 27cases seems awkward

Reply:
Please, remove it. This warning will not help you whether it is present or not.

Reply:
Fixed! Thanks!